src/install/installDeps.ts

```diff
@@ -28,7 +28,9 @@ export async function installDeps(browserTypes: string[]) {
   browserTypes.push('tools');
 
   const ubuntuVersion = await getUbuntuVersion();
-  if (ubuntuVersion !== '18.04' && ubuntuVersion !== '20.04') {
+  if (ubuntuVersion === '21.04') {
+    console.warn('WARNING: webkit does not work on Ubuntu 21.04');
```

Comment:
Let's print something more neutral (like '21 is not officially supported') as we are trying to launch webkit anyway.

Reply:
dropped it right away